libvast/src/format/test.cpp

```diff
@@ -176,7 +176,8 @@ struct randomizer {
   void operator()(const string_type&, std::string& str) {
     lcg gen{static_cast<lcg::result_type>(sample())};
     std::uniform_int_distribution<size_t> unif_size{0, 256};
-    std::uniform_int_distribution<char> unif_char{32, 126}; // Printable ASCII
+    std::uniform_int_distribution<signed char> unif_char{
```

Comment:
Note for the reviewer: `char` is not a valid integer type, and older clang and gcc versions were less compliant to the standard here and just allowed it. `signed char`, however, is a valid integer type, so that just works as a portable replacement.